### ISSUE TITLE
Play more with regularization

### ISSUE THREAD

**Issue by pdan93** [MantisAI/kaggle-feedback-2021#6]
A few ideas: 
- playing with the dropout rates
- https://virtual.2020.emnlp.org/paper_findings.178.html
- loss, optimizer and scheduler might have something to do with it

**Comment by pdan93** [comment]
Don't know if it goes here, but it's worth trying to freeze part of the model (tried it already but maybe we can experiment with it some mode)

**Comment by nsorros** [comment]
I would say loss, optimiser and scheduler would not have a lot to do with regularisation, only implicitly. In my mind regularisation main considerations are

- penalise the parameters i.e. dropout, l2, weight decay in general etc
- adjust capacity (size of model), used to be a thing, now less so as bigger models "scale" in the sense that they do not overfit and improve nowadays (miracle 🙏 )
- more data which is where data augmentation might play a role
- early stopping is definitely a form of regularisation

I would not focus too much on loss, optimiser, scheduler for regularisation
